test/test_torch.py

```diff
@@ -6546,6 +6546,40 @@ def test_unique_cuda(self):
             lambda: torch.unique(torch.cuda.FloatTensor([0., 1.])),
         )
 
+    def test_bincount_cpu(self):
+        # test weights
+        byte_counts = torch.bincount(torch.ByteTensor([0, 1, 1, 1, 4]),
+                                     torch.FloatTensor([.1, .2, .3, .4, .5]))
+        self.assertEqual(torch.DoubleTensor([0.1, 0.9, 0, 0, 0.5]), byte_counts)
+        # test tensor method without weights
+        long_counts = torch.LongTensor([0, 3, 2, 1, 3]).bincount()
+        self.assertEqual(torch.LongTensor([1, 1, 1, 2]), long_counts)
+        # test minlength functionality
+        int_counts = torch.bincount(torch.IntTensor([1, 1, 1, 1]), minlength=5)
```

Comment:
Don't the tests where `minlength` isn't being passed (e.g.: L6555) already cover this? Declaration in `native_functions.yaml`:
```
func: _bincount(Tensor self, Tensor? weights={}, int64_t minlength=0) -> Tensor
```

Reply:
Oh right my bad. I was thinking something that let's it return empty tensors. So what if you pass in empty data tensor?

Reply:
It will `throw`. See [Bincount.cpp:L24](https://github.com/pytorch/pytorch/pull/6688/files#diff-c871f1ca58769c40403510f93a2c0bddR24). Nonetheless, given the declaration, it is required for the `Tensor` to be defined. I'm assuming empty data tensor means undefined data tensor. For eg:
```bash
>>> a = torch.ByteTensor()
>>> a

[torch.ByteTensor of size (0,)]

>>> torch.bincount(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: invalid argument 1: tensor must have one dimension at ../src/TH/generic/THTensorMath.c:715
```

Reply:
Empty tensor and undefined tensor are two different things. The former exists but is empty. The latter doesn't exist.